man/systemd.link.xml

```diff
@@ -60,8 +60,11 @@
   <refsect1>
     <title>[Match] Section Options</title>
 
-    <para>A link file is said to match a device if each of the entries in the [Match] section matches, or if
-    the section is empty. The following keys are accepted:</para>
+    <para>A link file is said to match a device if each of the entries in the
+    <literal>[Match]</literal> section matches. When a link file does not contain valid settings in
+    <literal>[Match]</literal> section, then the file may match all devices and
```

Comment:
I'd prefer to change "if each of the entries in the [Match] section matches" → "if all matches specified by the [Match] section are satisfied". We have some settings which undo other settings, so I don't like saying that "all" need to match.

may match → will match

Maybe add a hint here to use `OriginalName=*` too:
```xml
Hint: to avoid the warning and to make it clear that all interfaces shall be matched, add <programlisting>OriginalName=*</>.
```